### ISSUE TITLE
Incorrect Syntax highlight: class property marked as comment

### ISSUE THREAD

**Issue by sgon00** [Dart-Code/Dart-Code#1554]
## Environment:

OS: MacOS

![Screen Shot 2019-03-25 at 19 43 27](https://user-images.githubusercontent.com/1555508/54917506-ecbabc80-4f36-11e9-86ca-68fa7bb91bb0.png)

![Screen Shot 2019-03-25 at 19 44 15](https://user-images.githubusercontent.com/1555508/54917535-00feb980-4f37-11e9-9c64-8d7ce972481f.png)

## Behaviors

Check `itemBuilder: `:

* The incorrect behavior screenshot:

![Screen Shot 2019-03-25 at 19 43 33](https://user-images.githubusercontent.com/1555508/54917598-1e338800-4f37-11e9-9487-f6a50b42357c.png)

* Correct behavior if I add a newline btw the comment and the class property:

![Screen Shot 2019-03-25 at 19 45 51](https://user-images.githubusercontent.com/1555508/54917621-28558680-4f37-11e9-9251-8d5e23f64ab8.png)

**Comment by DanTup** [comment]
I tried to copy what's in your screenshot, but couldn't repro:

![Screenshot 2019-03-25 at 9 26 28 am](https://user-images.githubusercontent.com/1078012/54918184-7f3a6c00-4ef5-11e9-9d15-561252755061.png)

Could you create a small complete file that triggers the issue and paste it here entirely so I can try with that?

**Comment by sgon00** [comment]
@DanTup Thank you very much for your quick reply. Please check the following updates:

* The incorrect line: 

```dart
child: Text('hello'),
```

* Screenshot

![Screen Shot 2019-03-25 at 20 58 19](https://user-images.githubusercontent.com/1555508/54921460-c863dd80-4f40-11e9-9df7-832cbf7be589.png)

* The code:

```dart
import 'package:flutter/material.dart';

void main() => runApp(App());

class App extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: "test",
      theme: ThemeData(primarySwatch: Colors.blueGrey),
      home: Scaffold(
        appBar: AppBar(
          elevation: 0.0,
        ),
        body: Container(
          /// comment
          child: Text('hello'), /// comment
        ),
      ),
    );
  }
}
```

**Comment by DanTup** [comment]
@sgon00 thanks, I can repro now! 👍

**Comment by DanTup** [comment]
This was caused by the begin/while regex for multi-line comments not anchoring the `///` to the start of the line for continuations (introduced by #1326). Fixed for vNext, though unfortuantely not in the beta version that I just pushed. I might push an updated beta if anything else urgent comes in in the next day or so.

Before:

![Screenshot 2019-03-25 at 1 10 12 pm](https://user-images.githubusercontent.com/1078012/54922114-66cf4f00-4eff-11e9-8694-573ef5ec257b.png)

After:

![Screenshot 2019-03-25 at 1 10 20 pm](https://user-images.githubusercontent.com/1078012/54922115-66cf4f00-4eff-11e9-8b1d-7798560351c5.png)
